cloudevents/tests/test_http_events.py

```diff
@@ -322,3 +323,32 @@ def test_valid_structured_events(specversion):
         assert event["source"] == f"source{i}.com.test"
         assert event["specversion"] == specversion
         assert event.data["payload"] == f"payload-{i}"
+
+
+@pytest.mark.parametrize("specversion", ["1.0", "0.3"])
+def test_create_binary_image(specversion):
+    size = (8, 8)
```

Comment:
What is size? What are we doing here?

Can we add a short comment as to what we'll do in this method?

Reply:
Removed size and added image_expected_shape to test we got the correct image

Reply:
I'm not seeing the changes made. Did you push them?